www-static/client/connect.css

```diff
@@ -68,6 +68,11 @@ tfoot {
   fill: #444;
 }
 
+.jfk-button {
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
```

Comment:
Why only top corners?

Reply:
The JFK standard buttons have a radius on all four corners.  However, the buttons on the connect page are at the top of the screen, so we want the top corners to be square, and leave the bottom corners be radiused.